### PR TITLE
Unify build fs layout

### PR DESCRIPTION
**Pull request by jakolehm**
Simplifies things.

### CODE REVIEW

File: src/common/vars/static-files-directory.injectable.ts [hunk 1]
```diff
@@ -4,16 +4,14 @@
...
 
 const staticFilesDirectoryInjectable = getInjectable({
   id: "static-files-directory",
 
   instantiate: (di) => {
     const joinPaths = di.inject(joinPathsInjectable);
-    const lensResourcesDir = di.inject(lensResourcesDirInjectable);
 
-    return joinPaths(lensResourcesDir, "static");
+    return joinPaths(__dirname, "..");
```
Comment (Iku-turso) [inline src/common/vars/static-files-directory.injectable.ts]: Side-effect :)

Comment (jakolehm) [reply src/common/vars/static-files-directory.injectable.ts]: Right, you mean `__dirname` here? Just wondering what would be the proper injectable to get the same result.

Comment (jakolehm) [reply src/common/vars/static-files-directory.injectable.ts]: Fixed.

File: src/common/vars/static-files-directory.global-override-for-injectable.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getGlobalOverride } from "../test-utils/get-global-override";
+import staticFilesDirectoryInjectable from "./static-files-directory.injectable";
+
+
+export default getGlobalOverride(staticFilesDirectoryInjectable, () => "some-static-directory");
```
Comment (Nokel81) [inline src/common/vars/static-files-directory.global-override-for-injectable.ts]: This needs to be an absolute path

**Review by Nokel81 — CHANGES_REQUESTED** [review]

File: src/common/vars/static-files-directory.global-override-for-injectable.ts (hunk 2 above)
Comment (Nokel81) [reply src/common/vars/static-files-directory.global-override-for-injectable.ts]: It could just be `/some-static-directory`

Comment (jakolehm) [reply src/common/vars/static-files-directory.global-override-for-injectable.ts]: Fixed.

**Review by Nokel81 — APPROVED** [review]

**Review by Nokel81 — APPROVED** [review]

**Pull request merged**